content/password.md

```diff
@@ -114,7 +114,7 @@ The output may be different based on the drive setup and partition table. You ca
 sudo cryptsetup luksDump /dev/sda3
 ```
 
-Replacing '/dev/sda3' with the location of the root partition on your system. With the output of this command we can see the seven extra slots that we have for passwords to decrypt the drive. Slots 1-7 are the open ones and Slot 0 is the current one the system has set.
+Replacing '/dev/sda3' with the location of the root partition on your system. With the output of this command we can see the seven extra slots that we have for passwords to decrypt the drive. On a normal install Slots 1-7 are the open, with Slot 0 as the initial pass-phrase.
```

Comment:
This is a bit odd:

On a normal install Slots 1-7 are the open, with Slot 0 as the initial pass-phrase.

I feel like it would be like this:

On a normal install Slots 1-7 are not used and Slot 0 is used as the initial pass-phrase.